### PR TITLE
Renamed page, added new section for updating private CA cert to non-private cert

### LINKED ISSUE #3680: [Rancher2] Updating a Private CA Certificate should be Updating the Rancher Certificate
**Request Summary:**

The docs [Updating a Private CA Certificate](https://rancher.com/docs/rancher/v2.5/en/installation/resources/update-ca-cert/) should be renamed to **Updating the Rancher Certificate** and have 2 main sections:
1. Updating a Private CA Certificate
_The current docs_

2. Updating from a Private CA Certificate to a Common Certificate
This new section should make it clear that it is possible to do the reverse procedure: change from a private certificate to a common (non private) certificate.

Some peculiarities of this scenario must be present on this new section:
- Secret `tls-ca` is no longer needed
- On rancher upgrade, the parameter `privateCA` **must be** either removed or set to `false`.
- The `CATTLE_CA_CHECKSUM` environment variable on downstream clusters agents **must be** updated to `""` (empty string)

**Details:**

### PR DESCRIPTION
**Pull request by jtravee**
Closes #3680

### CODE REVIEW

**Review by btat — CHANGES_REQUESTED** [review]
The installation/resources/tls-secrets/_index.md page has a link to this page that needs to be updated.

File: content/rancher/v2.5/en/installation/resources/update-rancher-cert/_index.md [hunk 1]
```diff
@@ -145,3 +146,13 @@ First, generate the agent definitions as described here: https://gist.github.com
...
+
+# Updating from a Private CA Certificate to a Common Certificate
+
+It is possible to perform the opposite procedure as shown above: you may change from a private certificate to a common, or non-private, certificate.
+
+The following conditions must be met to perform this process correctly:
+
+- The secret `tls-ca` is no longer needed.
+- On Rancher upgrade, the `privateCA` parameter **must** be either removed or set to `false`.
+- The `CATTLE_CA_CHECKSUM` environment variable on the downstream cluster agents **must** be updated to "" (an empty string).
```
Comment (thedadams) [inline content/rancher/v2.5/en/installation/resources/update-rancher-cert/_index.md:158]: In order to make the documentation consistent, I feel like these steps should be explicitly laid out like the "Updating a Private CA Certificate" steps. For example, in order to switch from a private to a trusted cert, the user will likely need to update the helm chart and go through similar steps as in "Reconfigure Rancher agents to trust the private CA" to ensure the `CATTLE_CA_CHECKSUM` is updated appropriately.

I would be happy to help lay out these steps if necessary.

**Review by thedadams — CHANGES_REQUESTED** [review]

File: content/rancher/v2.5/en/installation/resources/update-rancher-cert/_index.md (hunk 1 above)
Comment (jtravee) [reply content/rancher/v2.5/en/installation/resources/update-rancher-cert/_index.md:158]: @thedadams, that would be great if you could!

**Comment by jtravee** [comment]
Closed in favor of [#4119](https://github.com/rancher/docs/pull/4119).

**Pull request closed without merging**